template/client/store/actions.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+
+import * as mutationTypes from '~/types/mutations'
+import type {
```

Comment:
Could you please reorganize all imports like so: `import type { StateType } from '~/types'`?

The main idea is: if it fits in 80 chars it should be on a single line.